### ISSUE TITLE
Senior Backend Engineer

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#8362]
Senior Backend Engineer<br>
https://ift.tt/2KhstS8<br>
<p><img src="https://we-work-remotely.imgix.net/logos/0006/7361/logo.gif?ixlib=rails-2.1.3&amp;w=190&amp;min-h=150" alt="Logo.gif?ixlib=rails 2.1"></p>
<p><strong>Headquarters:</strong> San Francisco, CA<br>
<strong>URL:</strong> <a href="https://dataquest.breezy.hr/p/9f6b02bf5fec-senior-backend-engineer?source=weworkremotely">https://dataquest.breezy.hr/p/9f6b02bf5fec-senior-backend-engineer?source=weworkremotely</a></p>
<div>At&nbsp;<a href="https://dataquest.io/">Dataquest</a>, we teach data science to a community of hundreds of thousands of students worldwide. We help students deeply learn concepts, build projects, work together, and gain the confidence they need to start new careers. Read more about our philosophy&nbsp;<a href="https://www.dataquest.io/blog/the-perfect-data-science-learning-tool/">here</a>, and read what students think&nbsp;<a href="https://www.switchup.org/bootcamps/dataquest">here</a>.</div>
<div><br></div>
<div>We want to eventually help tens of millions of students worldwide change their lives through data science education, and you&rsquo;ll play a key role in helping us get there. We're looking for a Senior Backend Engineer to help support and evolve our learning platform.</div>
<div><br></div>
<div><span>Who Should Apply?</span></div>
<div><br></div>
<div>We currently need the most help in migrating our existing monolithic codebase to a microservices architecture, but you will also likely be jumping in on crafting a backend roadmap, maintaining and enhancing our existing application, and offering opinions on infrastructure. You must be passionate about learning, and enjoy developing new features without regressions. We&rsquo;re seeking someone with senior level experience who enjoys mentoring other engineers. You&rsquo;ll have a lot of autonomy in how you work, and will be able to set backend development guidelines organization-wide.</div>
<div><br></div>
<div>This is a fully remote role and we are open to applications worldwide.</div>
<div><br></div>
<div><span>How You&rsquo;ll Contribute:</span></div>
<div><br></div>
<ul>
<li>Lead the development of backend features</li>
<li>Help create an improved backend architecture</li>
<li>Set backend development guidelines and code quality standards</li>
<li>Mentor other backend engineers</li>
<li>Collaborate with team leads to ensure site uptime and reliability</li>
<li>Collaborate with DevOps to ensure new features are deployed reliably</li>
</ul>
<div><br></div>
<div><span>Requirements</span></div>
<div><br></div>
<div><span>Must Have:</span></div>
<div><br></div>
<ul>
<li>5 + years of web development experience</li>
<li>2 + years of Python and Django experience</li>
<li>Strong verbal and written communications skills -- we&rsquo;re a 100% remote team</li>
<li>Agile/SCRUM experience</li>
<li>Experience measuring the effectiveness of the features you work on</li>
<li>Experience leading architectural discussions and creating roadmaps</li>
<li>Experience mentoring other engineers</li>
<li>Experience with CI/CD pipelines</li>
</ul>
<div><br></div>
<div><span>Nice To Have:</span></div>
<div><br></div>
<ul>
<li>If a merge request deletes more code than it creates, you think it's a good day.</li>
<li>You aim to improve reliability and performance when you refactor.</li>
<li>You&rsquo;re eager to ask questions if you don't understand something and initial research isn't helping.</li>
<li>If you're blocked on something, you instinctively find other things to contribute while you wait.</li>
<li>When you hear about a new technology, you wonder how it can improve your work rather than worrying about having to learn it.</li>
</ul>
<div><br></div>
<div><span>Our Stack</span></div>
<div><br></div>
<div>These are some of the technologies you'll be working with at Dataquest, though you may end up introducing new ones:</div>
<div><br></div>
<ul>
<li>Our frontend is a modern React/Redux stack.</li>
<li>Communication with the backend happens over a combination of websockets, GraphQL, and REST APIs.</li>
<li>The main backend is Django on Python 3.</li>
<li>We use Docker extensively, both for student code execution, and for deployment of our services.</li>
<li>We use Kubernetes on Google Cloud to orchestrate our containers in production.</li>
<li>Our data is largely stored in Postgres, with some caching in Redis.</li>
<li>We use git for source control and CircleCI for build pipelines.</li>
</ul>
<div><br></div>
<div><span>Why you should work at Dataquest</span></div>
<div><br></div>
<div>At Dataquest, you'll get:</div>
<div><br></div>
<ul>
<li>The opportunity to see your impact on the lives of our students every day.</li>
<li>The chance to work with an engaged and mission-driven team.</li>
<li>The chance to contribute to product direction and decisions.</li>
<li>The ability to work remotely.</li>
<li>A supportive environment where you can get help when you run into blockers.</li>
</ul>
<p><strong>To apply:</strong> <a href="https://dataquest.breezy.hr/p/9f6b02bf5fec-senior-backend-engineer?source=weworkremotely">https://dataquest.breezy.hr/p/9f6b02bf5fec-senior-backend-engineer?source=weworkremotely</a></p>
<br>
<br>
via Remote Jobr Programming Jobs https://ift.tt/1WipnMk<br>
October 04, 2018 at 07:55AM
